cmd/erc20/transfer.go

```diff
@@ -49,8 +45,9 @@ func transferCmd() *cobra.Command {
 	return cmd
 }
 
-func transfer(ctx context.Context, rpcAddress string, mnemonic, ics20BankAddress, ics20TransferBankAddress string, fromIndex uint32, toAddress string, amount int64, denom string) error {
-	chain, err := geth.InitializeChain(ctx, rpcAddress, mnemonic, denom, ics20TransferBankAddress, ics20BankAddress)
+func transfer(ctx context.Context, rpcAddress string, mnemonic string, fromIndex uint32, toAddress string, amount int64, denom string) error {
+	dummyAddress := "0x0"
+	chain, err := geth.InitializeChain(ctx, rpcAddress, mnemonic, denom, dummyAddress, dummyAddress)
```

Comment:
This is a bit hacky.
Why not remove contract accessors (e.g. ERC20Token) from Chain struct?
A contract accessor can be easily created if needed.

Reply:
https://github.com/datachainlab/ethereum-ics20-cli/pull/8/commits/4b1995816bcbb49437d726a7626938cb731c8393
Removed tokenAddress from NewChain and AddXXX func

Reply:
Sorry for the confusion ... but I suggested that you should simplify the definition of Chain like below.
```
type Chain struct {
	chainID        int64
	mnemonicPhrase string
	keys           map[uint32]*ecdsa.PrivateKey
	Client        *client.ETHClient
}
```

Instantiation of `Erc20Token`, `ICS20Transfer` and `ICS20Bank` is very simple, and so it should be done at the location of use like below.
```
erc20Token, err := erc20.NewErc20(common.HexToAddress(tokenAddress), chain.Client)
if err != nil {
    <... error handling ...>
}
```

Reply:
If I have overlooked anything, please let me know.

Reply:
Sorry, my understanding was wrong.
I have corrected it.
https://github.com/datachainlab/ethereum-ics20-cli/pull/8/commits/563115db80d27c9629c2eed36cc2cebd80efb231